public/app/features/explore/Logs/LogsNavigation.tsx

```diff
@@ -110,6 +114,13 @@ function LogsNavigation({
         reportInteraction('grafana_explore_logs_pagination_clicked', {
           pageType: 'olderLogsButton',
         });
+
+        if (infiniteScrollWithOldersLogsButton) {
+          loadMoreLogs({ from: visibleRange.from - rangeSpanRef.current, to: visibleRange.from });
```

Comment:
I am a bit unsure about the foreseen behavior. With this, we might easily end in a spot were users change the timerange with the "Older logs" button, but will ultimately get out of their time range.

I don't think this is the foreseen behavior, or is it?

https://github.com/user-attachments/assets/c5670161-d88f-47a3-9783-1a977fb6f9a4

Reply:
That's a great point. I think we'll need to change the approach then.

Reply:
I moved the code out of LogsNavigation (which at this point is a deprecated component), to the InfiniteScroll one, where we have logic to know if we're within the current time selection. At least for Explore, this will round up the Infinite Scroll implementation to allow loading of older logs when the sorting order is ascending.